programs/drip/src/actions/admin.rs

```diff
@@ -127,6 +131,32 @@ impl<'a, 'info> Validatable for Admin<'a, 'info> {
                     DripError::VaultTokenAAccountIsEmpty
                 );
             }
+            Admin::AdminWithdraw { accounts } => {
+                validate!(
+                    accounts.admin.key() == accounts.vault_proto_config.admin,
+                    DripError::SignerIsNotAdmin
+                );
+
+                validate!(
+                    accounts.vault_proto_config.key() == accounts.vault.proto_config,
+                    DripError::InvalidVaultProtoConfigReference
+                );
+
+                validate!(
+                    accounts.vault_token_account.owner.key() == accounts.vault.key(),
+                    DripError::IncorrectVaultTokenAccount
+                );
```

Comment:
Should validate this is the token account specified in the vault since anyone can create a token account for the vault

Reply:
Hmm but I suppose its fine in this context though